lisa/analysis/pixel6.py

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# Copyright (C) 2015, ARM Limited and contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pandas as pd
+import holoviews as hv
+from holoviews import opts
+
+from lisa.datautils import df_refit_index
+from lisa.analysis.base import TraceAnalysisBase
+from lisa.trace import requires_events
+from lisa.notebook import plot_signal
+
+P6_EMETER_CHAN_NAMES = {
+    'S4M_VDD_CPUCL0': 'little',
+    'S3M_VDD_CPUCL1': 'mid',
+    'S2M_VDD_CPUCL2': 'big',
+}
+
+class Pixel6Analysis(TraceAnalysisBase):
+    """
+    Support for Pixel 6-specific analysis data
+
+    :param trace: input Trace object
+    :type trace: :class:`trace.Trace`
```

Comment:
s/:class:`trace.Trace`/lisa.trace.Trace/

Reply:
Every analysis file apart from `load_tracking.py` from what I can see says `trace.Trace` so I just did that for consistency. Will update, sure.

Reply:
Thanks for mentioning it, it's broken in 2 ways:
* old versions of Sphinx used to not create a link if provided the `:class:...` part in that specific location. Even today, I'm not sure if e.g. ``list(:class:`trace.Trace`)`` would work.
* It creates a link to that: https://docs.python.org/3/library/trace.html#trace.Trace
which is not at all the right thing :)

I'll fix existing broken locations

Reply:
I dug a bit and it turned out this issue was introduced in 2017 in what is now lisa legacy, before it was sanely packaged into a Python package (rather than a folder to add to PYTHON_PATH). I guess it worked at that time given because of the wonky layout, even though Python 2 already had `trace.Trace`. It has been copy/pasted since then. Sometimes the past still haunts us :ghost: